test/utils.jl

```diff
@@ -62,3 +62,63 @@
   @test Meshes.roundcoords(p₂, digits=10) == p₁
   @inferred Meshes.roundcoords(p₁, digits=10)
 end
+
+@testitem "bentleyottmann" setup = [Setup] begin
+  digits = Int(4) # precision in number of digits
```

Comment:
```suggestion
  digits = 4 # precision in number of digits
```
This is not necessary, `Int` is the default type for integer literals in Julia.